src/js/readonly.js

```diff
@@ -20,6 +20,10 @@ export default {
             const empty = !node.value && !node.dataset.calculate && !setValue && !node.classList.contains( 'readonly-forced' );
 
             node.classList.toggle( 'empty', empty );
+
+            if( node.classList.contains( 'empty' ) ){
+                node.setAttribute( 'aria-hidden', 'true' );
+            }
```

Comment:
Doesn't this mean the attribute will remain there forever (even if it becomes non-empty)?

Also `if(empty)` would probably be faster.

Reply:
going to remove `aria-hidden`  attribute when not `empty` is false